packages/features/ee/workflows/components/WorkflowListPage.tsx

```diff
@@ -96,9 +98,9 @@ export default function WorkflowListPage({ workflows, profileOptions, hasNoWorkf
                         </div>
                       </div>
 
-                      <ul className="mt-1 flex flex-wrap space-x-2 sm:flex-nowrap ">
+                      <ul className="mt-2 flex flex-wrap space-x-2 sm:flex-nowrap ">
                         <li>
-                          <Badge variant="gray">
+                          <Badge size="lg" variant="gray">
```

Comment:
If we do that in workflows and routing forms (https://github.com/calcom/cal.com/pull/9417#issuecomment-1587033998) we should also do it in event-types